ktor-http/common/src/io/ktor/http/content/Multipart.kt

```diff
@@ -27,7 +27,7 @@ public sealed class PartData(public val dispose: () -> Unit, public val headers:
      * @property provider of content bytes
      */
     public class FileItem(
```

Comment:
may be it could be just merged with BinaryChannelItem?

Reply:
But does it make sense to have specific implementation for it? This property could be an extension on `BinaryChannelItem`
May be, even for some reason someone will want to have `BinaryItem` with file content.

I don't know all specifics but looks a little odd, that we have `BinaryItem`, `FileItem` and `BinaryChannelItem` with almost the same semantics but different.

Reply:
`BinaryItem` uses `Input` which is blocking and already deprecated in the `main` branch. I don't see a problem in having separate entity for files in multipart, even if it's almost the same. Having `originalFileName` property on `BinaryChannelItem` can be confusing.